.github/actions/deploy/action.yml

```diff
@@ -0,0 +1,11 @@
+name: Deploy
```

Comment:
Nitpicking: deploy seems to be quite generic, we don't know if it's the documentation or the package. Maybe we can change it for `publish-package` or something more explicit, what do you think?